src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -371,26 +409,62 @@ CoordinateMap<SourceFrame, TargetFrame, Maps...>::inverse_impl(
   return tnsr::I<T, dim, SourceFrame>(std::move(mapped_point));
 }
 
+// define type-trait to check for time-dependent jacobian
+namespace CoordinateMap_detail {
+CREATE_IS_CALLABLE(jacobian)
+template <class T1, typename T2>
```

Comment:
change class and typename to typename and typename  - Also use better names for T1 and T2